Neo4j.Driver/Neo4j.Driver.Tests/TransactionTests.cs

```diff
@@ -28,15 +31,43 @@ namespace Neo4j.Driver.Tests
 {
     public class TransactionTests
     {
+        private static Expression<Action<IConnection>> RunRollback => x => x.Run("ROLLBACK", null, null, false);
```

Comment:
The previous usage pattern was a bit more self describing about what is expected at each call, but we may give a try anyways.